### [CVE-2017-2592](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-2592)
![](https://img.shields.io/static/v1?label=Product&message=python-oslo-middleware&color=blue)
![](https://img.shields.io/static/v1?label=Version&message=%20python-oslo-middleware%203.19.1%20&color=brightgreen)
![](https://img.shields.io/static/v1?label=Version&message=%20python-oslo-middleware%203.23.1%20&color=brightgreen)
![](https://img.shields.io/static/v1?label=Version&message=python-oslo-middleware%203.8.1%20&color=brightgreen)
![](https://img.shields.io/static/v1?label=Vulnerability&message=CWE-532&color=brightgreen)

### Description

python-oslo-middleware before versions 3.8.1, 3.19.1, 3.23.1 is vulnerable to an information disclosure. Software using the CatchError class could include sensitive values in a traceback's error message. System users could exploit this flaw to obtain sensitive information from OpenStack component error logs (for example, keystone tokens).

### POC

#### Reference
No PoCs from references.

#### Github
- https://github.com/ARPSyndicate/cvemon

